db/seeds/sort_and_filter/jobs_seed.js

```diff
@@ -0,0 +1,67 @@
+exports.seed = function (knex) {
+  return knex('jobs').del() // Deletes ALL existing entries
+    .then(function () { // Inserts seed entries one by one in series
+      return knex('jobs').insert({
+        id: '100',
+        accountId: 'c8330c1f-23f5-4577-943d-151d059af588',
+        title: 'Organ Grinder',
+        employer: 'Monkey',
+        created_at: '2016-06-29 14:45:27',
+        updated_at: '2016-06-29 14:45:27',
+        sourceType: 'online',
+        sourceUrl: 'http://www.stuff.com',
+        deadline: '2016-07-14',
+        rating: '1',
+        status: 'applied',
+        status_sort_index: 1,
+      });
```

Comment:
we could extract jobs array to `fixtures.js` in order to have access to them in tests